frontend/src/components/TreeViewer/TreeViewer.jsx

```diff
@@ -305,7 +377,6 @@ export const TreeViewer = ({ data }) => {
       clickedNode.data.name,
       curViewingNodeID.current
     );
-    //previouslyClickedNode.current = clickedNode.data.name;
     const nodeID = getParameterByName("id", clickedNode.data.url);
     const nodeInfoUrl = replaceSpaceCharacters(
       `http://localhost:3003/get-node-info/${nodeID}`
```

Comment:
We should probably add the base address here ("http://localhost:3003") and all other uses of it to the config file

Reply:
Seems like this is out of scope for this MR; instead we can address it in the other cleanup MR here: https://github.com/VerWiki/VerWikiApp/pull/21